From 0970909e987782f91c7eb887df04c7209779d2a2 Mon Sep 17 00:00:00 2001
From: garypang13 <48883331+garypang13@users.noreply.github.com>
Date: Tue, 13 Oct 2020 02:43:20 +0800
Subject: [PATCH] Update smartdns

---
 package/openwrt/files/etc/init.d/smartdns | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/package/openwrt/files/etc/init.d/smartdns b/package/openwrt/files/etc/init.d/smartdns
index 2bf799e..5f8bd86 100644
--- a/package/openwrt/files/etc/init.d/smartdns
+++ b/package/openwrt/files/etc/init.d/smartdns
@@ -297,6 +297,11 @@ load_service()
 	config_get serve_expired "$section" "serve_expired" "0"
 	[ "$serve_expired" = "1" ] && conf_append "serve-expired" "yes"
 
+	config_get anti_ad "$section" "anti_ad" "0"
+	[ "$anti_ad" = "1" ] && {
+		conf_append "conf-file" "$SMARTDNS_VAR_CONF_DIR/anti-ad-smartdns.conf"
+	}
+
 	SMARTDNS_PORT="$port"
 
 	config_get cache_size "$section" "cache_size" ""
@@ -387,4 +392,9 @@ reload_service()
 {
 	stop
 	start
+	[ "$anti_ad" = "1" ] && {
+		wget -O $SMARTDNS_VAR_CONF_DIR/anti-ad-smartdns.conf https://anti-ad.net/anti-ad-for-smartdns.conf || true
+	stop
+	start
+	}
 }
