From 75e28c7bada05ac7bd26d55b7a10f55bc5fda139 Mon Sep 17 00:00:00 2001
From: garypang13 <48883331+garypang13@users.noreply.github.com>
Date: Sun, 11 Oct 2020 18:04:50 +0800
Subject: [PATCH] Update smartdns

---
 package/openwrt/files/etc/init.d/smartdns | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/package/openwrt/files/etc/init.d/smartdns b/package/openwrt/files/etc/init.d/smartdns
index 2bf799e..e48b587 100644
--- a/package/openwrt/files/etc/init.d/smartdns
+++ b/package/openwrt/files/etc/init.d/smartdns
@@ -276,16 +276,16 @@ load_service()
 	config_get ipv6_server "$section" "ipv6_server" "1"
 	config_get tcp_server "$section" "tcp_server" "1"
 	if [ "$ipv6_server" = "1" ]; then
-		conf_append "bind" "[::]:$port"
+		conf_append "bind" "[::]:$port  -no-speed-check"
 	else
-		conf_append "bind" ":$port"
+		conf_append "bind" ":$port  -no-speed-check"
 	fi
 
 	[ "$tcp_server" = "1" ] && {
 		if [ "$ipv6_server" = "1" ]; then
-			conf_append "bind-tcp" "[::]:$port"
+			conf_append "bind-tcp" "[::]:$port  -no-speed-check"
 		else
-			conf_append "bind-tcp" ":$port"
+			conf_append "bind-tcp" ":$port  -no-speed-check"
 		fi
 	}
 	config_get dualstack_ip_selection "$section" "dualstack_ip_selection" "0"
